examples/react/index.html

```diff
@@ -14,6 +12,9 @@
   <!-- Insert any 3rd-party scripts below-->
   <!-- <script src=""></script >-->
   <div id="root"></div>
-  <script src="script-compiled.js"></script>
+  <script src="babel-standalone-7.11.6.min.js"></script>
+  <script src="react-16.13.1.min.js"></script>
+  <script src="react-dom-16.13.1.min.js"></script>
+  <script type="text/jsx" src="script.js"></script>
```

Comment:
Thanks, this makes this demo a lot more like the other ones instead of being a weird exception that requires its own file watching and build and stuff.

Reply:
Yeah, I think it was a good idea on your part. 😄